mkdocs/topics/linting.md

```diff
@@ -1,20 +1,22 @@
-# Linting `visivo_project.yml`
+# Linting `project.visivo.yml`
 
-We publish a json schema file for our `visivo_project.yml`.  This gives you realtime feedback when creating your project.  Currently is it available at https://docs.visivo.io/assets/visivo_schema.json.
+We publish a json schema file for our `project.visivo.yml`.  This gives you realtime feedback when creating your project.  Currently is it available at https://docs.visivo.io/assets/visivo_schema.json and is published SchemaStore.
 
 ## VSCode Plugin
 
 The YAML plugin by redhat-developer [here](https://github.com/redhat-developer/vscode-yaml)
  can lint yaml with a json schema.
 
-### Configuration
+### Configuration (Optional)
+
+By default any file with the format `*.visivo.yml` will be linted.  If you have a file that is named differently, you can add it to the local configuration.
```

Comment:
Does this mean that you don't need to edit the yaml schema at all if you are using the *.visivo.yml naming convention?

Reply:
That is correct!